src/kubernetes.cc

```diff
@@ -115,6 +117,50 @@ KubernetesReader::KubernetesReader(const Configuration& config,
                                    HealthChecker* health_checker)
     : config_(config), environment_(config), health_checker_(health_checker) {}
 
+MetadataStore::Metadata KubernetesReader::GetMetadataOnly(
+    const json::Object* resource, Timestamp collected_at, bool is_deleted) const
```

Comment:
Is this what they're called? "Kubernetes resources"? I thought they were "Kubernetes objects"...

Reply:
renamed.